### PR TITLE
fixup! platform: Add support for new SOCs: nRF5340 and nRF9160

### PR DESCRIPTION
**Pull request by oyvindronningstad**
Will be squashed

Fix copyright headers and remove ARMCLANG and IAR support (since it
was untested).

Signed-off-by: Øyvind Rønningstad <oyvind.ronningstad@nordicsemi.no>

### CODE REVIEW

**Comment by oyvindronningstad** [comment]
cc @ioannisg @anangl

**Review by ioannisg — APPROVED** [review]
Looks good to me
Seems only cleanup, right?

File: platform/ext/target/nordic_nrf/common/CMakeLists.txt [hunk 1]
```diff
@@ -8,27 +9,22 @@
 cmake_policy(SET CMP0076 NEW)
 set(CMAKE_CURRENT_SOURCE_DIR ${CMAKE_CURRENT_LIST_DIR})
 
+
```
Comment (anangl) [inline platform/ext/target/nordic_nrf/common/CMakeLists.txt]: This extra line does not seem to be needed.

**Review by anangl — APPROVED** [review]

**Comment by oyvindronningstad** [comment]
> Seems only cleanup, right?

Sure. Glad to have some eyes on the copyright changes though.

**Pull request merged**